db/database.go

```diff
@@ -1500,6 +1506,10 @@ func (db *Database) Compact(ctx context.Context, skipRunningStateCheck bool, cal
 	}
 	base.InfofCtx(ctx, base.KeyAll, "Finished compaction of purged tombstones for %s... Total Tombstones Compacted: %d", base.MD(db.Name), purgedDocCount)
 
+	if purgeErrorCount > 0 || deleteErrorCount > 0 || addErrorCount > 0 {
+		base.ErrorfCtx(ctx, "compaction finished with %d add key errors, %d delete key errors and %d purge key errors", addErrorCount, deleteErrorCount, purgeErrorCount)
```

Comment:
I wonder if this should just be warn. Don't think there's anything a user can do about it really other than trying compaction again (which will be on a scheduled task anyway) 

```suggestion
		base.WarnfCtx(ctx, "compaction finished with %d add key errors, %d delete key errors and %d purge key errors", addErrorCount, deleteErrorCount, purgeErrorCount)
```

Reply:
Agreed, will make that change!